_commands/basics/cd.md

```diff
@@ -16,7 +16,87 @@ $ cd /home
 
 ### Useful Options / Examples
 
+
 #### `cd -`
```

Comment:
I just pushed the updated file. One thing I wasn't able to figure out is how to actually create a hyperlink out of a url in the text file, so the link doesn't show up blue and can't be followed with a click. I know this is a small detail but I can go ahead and modify that too, since it would make the page look nicer, given that it is possible to do so.